.github/workflows/ci.yml

```diff
@@ -30,8 +30,7 @@ jobs:
     strategy:
       matrix:
         NXF_VER:
-          - "24.04.2"
-          - "latest-everything"
```

Comment:
No more tests on the latest version?

Reply:
eh 🤷🏻  it's all overhead for not much profit

Reply:
Allright! as long as it works on the version we are using :p